### ISSUE TITLE
WIFI Adapter Issues since installing 2.0.0

### ISSUE THREAD

**Issue by tdw46** [tjackenpacken/taskbar-groups#128]
Is there any way your app could be causing this issue? I don't see any crazy network draw in task manager while running your app, but maybe the issue is something you have seen before?

Basically, after anywhere between 5 minutes and 24 hours of restarting my computer, the wifi adapter powers off and device manager shows that it cannot be powered back on. So I _must_ restart my computer to get it to work again.

![Taskbar Groups Wifi Fail Screenshot](https://user-images.githubusercontent.com/10996763/118339457-e73dd900-b4e6-11eb-8466-7d1e0602ef82.png)

**Comment by anas6012** [comment]
some apps i can not open it on the Taskbargroup for example google chrome and office 365

**Comment by PikeNote** [comment]
I don't believe so. We don't interface with the drivers in any way possible.
I assume you aren't using the application to open anything, so I don't believe that is the issue.
I will keep a lookout, but I don't believe this is causing that issue. It may be a physical or driver failure.

See if a new driver from [here](https://downloadcenter.intel.com/product/99446/Intel-Wireless-AC-9560) will help.
